Source/i_video.c

```diff
@@ -896,7 +888,7 @@ int widescreen; // widescreen mode
 
 void I_FinishUpdate(void)
 {
-   if (noblit || !in_graphics_mode)
+   if (!in_graphics_mode)
       return;
```

Comment:
I think the `noblit` mode may even prove useful. In contrast to `nodraw` which doesn't even call Doom's drawing functions, this only omits the rendering from framebuffer to screen. So, if you let `noblit` and `nodraw` run against each other, you'll find out how much time you spend in SDL's rendering part.